packages/java/maven-plugin/src/main/java/dev/hilla/maven/PluginConfiguration.java

```diff
@@ -0,0 +1,69 @@
+package dev.hilla.maven;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class PluginConfiguration {
+    public static final String RESOURCE_NAME = "hilla-generator-configuration.json";
```

Comment:
```suggestion
    public static final String RESOURCE_NAME = "hilla-engine-configuration.json";
```

BTW should we name it with “engine”, as it combines parser and generator configuration?